iosHyperskillApp/iosHyperskillApp/Sources/Frameworks/Notifications/UserNotificationsCenterDelegate.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import UserNotifications
+
+final class UserNotificationsCenterDelegate: NSObject {
+    func attachNotificationDelegate() {
+        UNUserNotificationCenter.current().delegate = self
+    }
+}
+
+extension UserNotificationsCenterDelegate: UNUserNotificationCenterDelegate {
+    func userNotificationCenter(
+        _ center: UNUserNotificationCenter,
+        willPresent notification: UNNotification,
+        withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void
+    ) {
+        completionHandler([.alert, .sound])
```

Comment:
`completionHandler([.banner, .sound])`, `.alert` Deprecated since iOS 14 https://developer.apple.com/documentation/usernotifications/unnotificationpresentationoptions/1649506-alert